src/foraging_gui/Foraging.py

```diff
@@ -369,6 +367,8 @@ def _check_drop_frames(self,save_tag=1):
 
             camera_trigger_file=os.path.join(HarpFolder,'BehaviorEvents','Event_94.bin')
             if os.path.exists(camera_trigger_file):
+                # sleep some time to wait for the finish of saving video
+                time.sleep(5)
```

Comment:
How do you know that 5 seconds is enough? Is there a way to tell when the saving is done? (at least you can check the file size until it stops growing)

Reply:
From what I observed, when we stop logging for the current codec, the save completes almost immediately (within 2 seconds). I set it long (5s) to make it safer.